### ISSUE TITLE
Sats for review

### ISSUE THREAD

**Issue by adamjonas** [bitcointranscripts/transcription-review-front-end#16]
As a reviewer, I will be paid 1 sat per word once my reviewed transcript is merged into bitcointranscripts. This is credited to my user account and can be withdrawn over lightning at any time.  This requires:  A hook to track whether a PR opened by a reviewer has been merged from the review app The reviewers to auth with a LN-URL service that has ln-url withdraw support (https://bolt.fun/guide/web-services/lnurl/withdraw) or to generate their own invoice for the sats and copy and paste it into an input box like https://stacker.news/wallet?type=withdraw
